windows/utils/win_enable_vss_log.yml

```diff
@@ -35,3 +35,8 @@
 - include_tasks: win_execute_cmd.yml
   vars:
     win_powershell_cmd: "Restart-Service -Force -Name VMTools"
+
+# Make sure VMware tools service is running before taking quiesce snapshot
+- include_tasks: win_wait_service_status.yml
```

Comment:
We have a common task common/vm_wait_vmtools_status.yml. Could it be used here?

Reply:
I'm afraid there is time slot between the VMware tools status got and VMware tools status refresh.